app/uk/gov/hmrc/nationalinsurancerecord/config/ApplicationConfig.scala

```diff
@@ -18,12 +18,18 @@ package uk.gov.hmrc.nationalinsurancerecord.config
 
 import com.google.inject.{Inject, Singleton}
 import play.api.Configuration
+import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
 
 @Singleton
-class ApplicationConfig @Inject()(configuration: Configuration) {
-  val responseCacheTTL = configuration.getInt("mongodb.responseTTL").getOrElse(throw new RuntimeException("MongoDB TTL is not configured"))
+class ApplicationConfig @Inject()(configuration: Configuration, servicesConfig: ServicesConfig) {
 
-  lazy val authorization: String = s"Bearer ${configuration.getString("microservice.services.des-hod.authorizationToken").getOrElse("Local")}"
+  val serviceUrl = servicesConfig.baseUrl("citizen-details")
```

Comment:
Suggestion

You could put a better variable name here to indicate the url is for cit details

Reply:
changed to val citizenDetailsUrl